examples/ci-tests/create-and-destroy-aws/service-docdb.yml

```diff
@@ -0,0 +1,28 @@
+environments:
```

Comment:
I don't see the benefit of separating config for each storage. Let's just have one yaml with everything